frontends/mit-open/src/page-components/ChannelDetails/ChannelDetails.tsx

```diff
@@ -0,0 +1,187 @@
+import { capitalize } from "ol-utilities"
+import { ChannelTypeEnum, FieldChannel } from "api/v0"
+import OpenInNewIcon from "@mui/icons-material/OpenInNew"
+
+type ChannelDetailsProps = {
+  field: FieldChannel
+}
+
+const FACETS_BY_CHANNEL_TYPE: Record<ChannelTypeEnum, string[]> = {
+  [ChannelTypeEnum.Topic]: [
+    "free",
+    "department",
+    "offered_by",
+    "learning_format",
+  ],
+  [ChannelTypeEnum.Department]: [
+    "free",
+    "topic",
+    "offered_by",
+    "learning_format",
+  ],
+  [ChannelTypeEnum.Offeror]: [
+    "offerings",
+    "audience",
+    "formats",
+    "content_types",
+    "certifications",
+    "more_information",
+  ],
+  [ChannelTypeEnum.Pathway]: [],
+}
+
+const getFacetManifest = (channelType: ChannelTypeEnum) => {
+  return [
+    {
+      type: "group",
+      facets: [
+        {
+          name: "free",
+          label: "Free",
+        },
+      ],
+      name: "free",
+    },
+    {
+      name: "topic",
+      title: "Topic",
+    },
+    {
+      name: "formats",
+      title: "Formats",
+    },
+    {
+      name: "fee",
+      title: "Fee",
+    },
+    {
+      name: "department",
+      title: "Department",
+    },
+    {
+      name: "offerings",
+      title: "Offerings",
+    },
+    {
+      name: "level",
+      title: "Level",
+    },
+    {
+      name: "content_types",
+      title: "Type of Content",
+    },
+    {
+      name: "audience",
+      title: "Audience",
+    },
+    {
+      name: "more_information",
+      title: "More Information",
+      labelFunction: (key: string, channelTitle: string) => (
+        <a href={key}>
+          {channelTitle} website <OpenInNewIcon fontSize="inherit" />
+        </a>
+      ),
+    },
+    {
+      name: "platform",
+      title: "Platform",
+    },
+    {
+      name: "offered_by",
+      title: "Offered By",
+    },
+    {
+      name: "certifications",
+      title: "Certificate",
+    },
+    {
+      name: "learning_format",
+      title: "Format",
+      labelFunction: (key: string) =>
+        key
+          .split("_")
+          .map((word) => capitalize(word))
+          .join("-"),
+    },
+  ].filter((facetSetting) =>
+    (FACETS_BY_CHANNEL_TYPE[channelType] || []).includes(facetSetting.name),
+  )
+}
+
+const getChannelDetails = (field: FieldChannel) => {
+  const channelType = field.channel_type
+  const dataKey = `${channelType}_detail`
+  const fieldData = field as unknown as Record<string, string[] | string>
+  const data = fieldData[dataKey] as unknown as Record<
+    string,
+    string[] | string
+  >
+  return data[channelType]
+}
+const InfoLabel = styled(Typography)(({ theme }) => ({
+  color: theme.custom.colors.mitRed,
+}))
+const ChannelDetailsCard = styled(Box)(({ theme }) => ({
+  border: "1px solid #ccc",
+  borderRadius: "8px",
+  backgroundColor: "white",
+  minWidth: "300px",
+  width: "50%",
+  fontSize: "0.5em",
+  lineHeight: "0.5em",
+  [theme.breakpoints.down("md")]: {
+    width: "100%",
```

Comment:
The panel isn't occupying its full space on mobile: 

<img width="467" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/2a83edb8-d7fb-45ba-8e04-b7117841a9b7">

Reply:
should look now